test/suites/ubuntu-core-edgex-image/test.sh

```diff
@@ -1,23 +1,39 @@
 #!/bin/bash -e
 
-ssh "$USER@localhost" -p 8022 "snap install curl"
+SSH_USER="$USER"
+SSH_PORT="8022"
 
-# check status of core services inside of the emulator
+printf "# Install curl on host:\n"
+snap install curl; echo
```

Comment:
You are trying too hard to make the output look pretty. I'd just keep descriptive comments and enable xtrace to print the commands. Tip: For adding newline to echo, use -e, e.g. `echo -e "first line \n second line"`